tests/cli/test_cli.py

```diff
@@ -103,3 +105,33 @@ def test_cli_convert_ome_tiff(grid_layout, tmpdir):
     result = runner.invoke(cli, cmd)
     assert result.exit_code == 0, result.output
     assert "Converting" in result.output
+
+
+def test_cli_update_scale_metadata():
```

Comment:
I recommend testing both the CLI (end-to-end testing) and the underlying API (unit testing).

Reply:
More concrete example: the function can be tested with a single FOV, and the CLI can be tested with a wild card pattern.

Reply:
Thanks @ziw-liu, I've revised this branch to include a CLI and API with tests on both.